apollo-audit/apollo-audit-impl/src/main/java/com/ctrip/framework/apollo/audit/aop/ApolloAuditSpanAspect.java

```diff
@@ -87,13 +89,18 @@ void auditDataInfluenceArg(ProceedingJoinPoint pjp) {
 
   Method findMethod(ProceedingJoinPoint pjp) {
     Class<?> clazz = pjp.getTarget().getClass();
+    Signature pjpSignature = pjp.getSignature();
     String methodName = pjp.getSignature().getName();
-    for (Method method : clazz.getDeclaredMethods()) {
-      if (method.getName().equals(methodName)) {
-        return method;
-      }
+    Class[] parameterTypes = null;
+    if (pjpSignature instanceof MethodSignature) {
+      parameterTypes = ((MethodSignature) pjpSignature).getParameterTypes();
+    }
+    try {
+      Method method = ReflectUtils.findDeclaredMethod(clazz, methodName, parameterTypes);
+      return method;
+    } catch (NoSuchMethodException e) {
+      return null;
```

Comment:
Is it necessary for the caller to implement handling mechanisms for null value scenarios?